### PR TITLE
Track host OS changes in GitHub actions

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jobs:
           script: ./gradlew connectedCheck
 
   ios:
-    runs-on: macos-latest
+    runs-on: macos-latest-large
```
Comment (JakeWharton) [inline .github/workflows/build.yaml]: Why switch this? Once code signing is disabled it should work fine on ARM, no?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ jobs:
 
   android-emulator:
     # We build on a Mac to get hardware acceleration for the Android emulator.
-    runs-on: macos-latest
+    runs-on: macos-latest-large
```
Comment (JakeWharton) [inline .github/workflows/release.yaml]: This should be moved to ubuntu which is faster than both architectures of macs

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -84,7 +84,7 @@ jobs:
           script: ./gradlew connectedCheck
 
   publish:
-    runs-on: macos-latest
+    runs-on: macos-latest-large
```
Comment (JakeWharton) [inline .github/workflows/release.yaml]: This can probably be arm

File: samples/world-clock/ios/app/WorldClock.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -284,6 +303,7 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				COPY_PHASE_STRIP = NO;
+				CODE_SIGNING_ALLOWED = NO;
```
Comment (JakeWharton) [inline samples/world-clock/ios/app/WorldClock.xcodeproj/project.pbxproj:306]: Ooo I was specifying this on the command line but I like it better in this garbage file where no one has to see it

**Review by JakeWharton — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (swankjesse) [reply .github/workflows/build.yaml]: Just attempting to preserve the old behavior. I think ARM is an improvement, I’ll switch

File: .github/workflows/release.yaml (hunk 2 above)
Comment (swankjesse) [reply .github/workflows/release.yaml]: Fixed.

File: .github/workflows/release.yaml (hunk 3 above)
Comment (swankjesse) [reply .github/workflows/release.yaml]: Fixed.

**Pull request merged**